src/DocumentDB.ChangeFeedProcessor/PartitionManagement/RequestOptionsFactoryForParitionedByIdColleciton.cs

```diff
@@ -0,0 +1,21 @@
+﻿//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  Licensed under the MIT license.
+//----------------------------------------------------------------
+
+namespace Microsoft.Azure.Documents.ChangeFeedProcessor.PartitionManagement
+{
+    using Microsoft.Azure.Documents.Client;
+
+    internal class RequestOptionsFactoryForParitionedByIdColleciton : IRequestOptionsFactory
```

Comment:
nit: mispell should be `RequestOptionsFactoryForPartitionedByIdCollection`. File should be renamed too.

```suggestion
    internal class RequestOptionsFactoryForPartitionedByIdCollection: IRequestOptionsFactory
```

Reply:
done, thanks!